src/screen/CalendarScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import moment from 'moment';
```

Comment:
Can we switch to a different date library such as `dayjs` or `date-fns` ..
moment is not actively developed anymore.

Reply:
done ✅